MassSpectrometry/MsDataFile.cs

```diff
@@ -183,14 +182,15 @@ public static void WindowModeHelper(ref double[] intensities, ref double[] mArra
                     tempIntList.Add(intensities[arrayIndex]);
                     tempMzList.Add(mArray[arrayIndex]);
                 }
-                if (filteringParams.WindowMaxNormalizationValue.HasValue)
+                if (filteringParams.NormalizePeaksAccrossAllWindows)
                 {
                     double max = tempIntList.Max();
                     if (max == 0)
                     {
                         max = 1;
                     }
-                    tempIntList = tempIntList.Select(x => x / max * filteringParams.WindowMaxNormalizationValue.Value).ToList();
+                    //currently normalizing all peaks to 100
+                    tempIntList = tempIntList.Select(x => x / max * 100d).ToList();
```

Comment:
Remove the 100 and just normalize it to the maximum peak being 1 (x => x / max).

Reply:
done